index.html

```diff
@@ -0,0 +1,388 @@
+            <br />
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </main>
+
+    <!-- Contact form start -->
+    <footer id="contact" class="footer">
+      <div class="form-container">
+        <p class="contact-msg">
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in
+          touch.
+        </p>
+        <form action="https://formspree.io/f/" method="POST" class="form">
+          <div class="form-control">
+            <input id="name" name="user_name" placeholder="Full Name" />
+          </div>
+          <div class="form-control">
+            <input id="email" name="user_email" placeholder="Email address" required />
+          </div>
+          <div class="form-control">
+            <textarea name="comment" maxlength="500" id="message" placeholder="Enter text here"></textarea>
+          </div>
+          <div class="submit-container">
+            <button type="submit" class="get-in-touch-btn card-see">Get in touch</button>
```

Comment:
- [ ] Your button design should have interactions such as hover, active (when it's pressed), disabled and enable (default state) styled as shown in the [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). You can implement those styles in your CSS file. Take a look at how it looks:
![Capture](https://user-images.githubusercontent.com/80612925/169068484-f9118a6a-a208-4fde-bdb6-b906bd49af5d.PNG)